src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.subway.ui;
+
+import java.net.URI;
+import java.util.List;
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LineController {
```

Comment:
사소하지만 해당 api들의 시작은 lines로 시작하고 있어서
위로 빼줘도 좋을 것 같아요 :)

```suggestion
@RestController
@RequestMapping("lines")
public class LineController {
```

Reply:
@ohtaeg 
다음 단계 진행할 때 같이 변경하도록 하겠습니다. 😄